source/guide_pelican.rst

```diff
@@ -0,0 +1,182 @@
+
+----
+
+License
+=======
+
+Pelican is free and open source software licensed under `AGPL 3 <https://www.gnu.org/licenses/agpl-3.0.en.html>`_.
+
+Prerequisites
+=============
+
+Pelican currently runs best on Python 3.6+; earlier versions are not supported. 
+This version comes preinstalled on your uberspace.
+
+.. note:: If you want to use your own website domain, refer to the uberspace manual for :manual:`domains <web-domains>`
+
+Installation
+============
+
+Pelican is installed simply by using pip3
+
+::
+
+ [isabell@stardust isabell]$ pip3 install --upgrade --user "pelican[markdown]"
+
+.. note:: If you prefer to install the latest bleeding-edge version of Pelican rather than a stable release, use the following command:
+
+  ::
+
+    pip3 install --upgrade --user -e "git+https://github.com/getpelican/pelican.git#egg=pelican"
+
+
+Generate your website
+=====================
+
+**Step 1:** 
+
+Pelican comes with a handy quickstart wizard that creates the basic structure for you.
+Run it inside a subfolder. In this guide, we'll use ``~/pelican``
+
+:: 
+
+  [isabell@stardust isabell]$ mkdir ~/pelican && cd ~/pelican
+  [isabell@stardust pelican]$ pelican-quickstart
+
+  Welcome to pelican-quickstart v4.6.0.
+
+  This script will help you create a new Pelican-based website.
+
+  Please answer the following questions so this script can generate the files
+  needed by Pelican.
+
+  > Where do you want to create your new web site? [.] <enter>
+  > What will be the title of this web site? my-shiny-blogtitle
+  > Who will be the author of this web site? isabell
+  > What will be the default language of this web site? [de] <enter>
+  > Do you want to specify a URL prefix? e.g., https://example.com   (Y/n) <enter>
+  > What is your URL prefix? (see above example; no trailing slash) https://isabell.uber.space
+  > Do you want to enable article pagination? (Y/n) <enter>
+  > How many articles per page do you want? [10] <enter>
+  > What is your time zone? [Europe/Paris] Europe/Berlin
+  > Do you want to generate a tasks.py/Makefile to automate generation and publishing? (Y/n) <enter>
+  > Do you want to upload your website using FTP? (y/N) <enter>
+  > Do you want to upload your website using SSH? (y/N) <enter>
+  > Do you want to upload your website using Dropbox? (y/N) <enter>
+  > Do you want to upload your website using S3? (y/N) <enter>
+  > Do you want to upload your website using Rackspace Cloud Files? (y/N) <enter>
+  > Do you want to upload your website using GitHub Pages? (y/N) <enter>
+  Done. Your new project is available at /home/isabell/pelican
+
+  [isabell@stardust isabell]$ 
+
+
+.. note:: In case you want to use a custom domain other than isabell.uber.space, you can enter it here or easily change it later in ``~/pelican/pelicanconf.py``
+
+**Step 2:**
+
+You need to modify ``~/pelican/Makefile`` to make sure the publish process is working correctly on your uberspace.
```

Comment:
Modifying the `OUTPUTDIR` varliable seems cleaner and easier to me, what do you think?
```
OUTPUTDIR=/var/www/virtual/$(USER)/html
```

Reply:
Hi Luto,

thanks for your review and suggestions.
I was about to implement the same, but there is a pelican [setting](https://docs.getpelican.com/en/stable/settings.html) named  "DELETE_OUTPUT_DIRECTORY"
From the docs:

> Delete the output directory, and all of its contents, before generating new files. This can be useful in preventing older, unnecessary files from persisting in your output. However, this is a destructive setting and should be handled with extreme care.

So I decided to go with the additional rsync as this allows to specify excludes like `--exclude /.htaccess`.

Reply:
Deleting everything seems fine to me. All guides in the lab should assume an empty uberspace. If you'd like, add a `.. warning::` block above the `make publish` step saying
> This step will delete all content at `~/html`, if any.

The default pelican step does not seem to generate a `.htaccess` file and the default uberspace does not come with one, so I am not quite sure which `.htaccess` you'd like to ignore or preserve?

Reply:
I have added a custom .htaccess for http-> https redirect and securtiy headers. 
I am fine with adding the suggested warning and change. My [commit](https://github.com/Uberspace/lab/pull/1003/commits/ad722b0b4cf44a90fdb4478e6bd4108ec377f350) was added before I read your last comment.

Reply:
I see! HTTP->HTTPS is handled by us globally, so no HTTP connection will ever reach your uberspace. Security headers should be set using [`web header`](https://manual.uberspace.de/web-headers/).